packages/terra-form-textarea/src/Textarea.module.scss

```diff
@@ -117,19 +117,19 @@
 
     &:focus {
       background-color: var(--terra-form-textarea-error-focus-background-color);
-      background-size: var(--terra-form-textarea-error-focus-keyboard-animation-size, 0 0);
+      background-size: var(--terra-form-textarea-error-focus-keyboard-animation-background-size, 0 0);
```

Comment:
I'd drop the words `keyboard-animation` from this variable to make it consistent with the other variables.

Reply:
Resolved in this commit: https://github.com/cerner/terra-core/pull/2125/commits/8a9967b6e0cf7ff56d7f2d657946e5c82e1c59fc

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/8a9967b6e0cf7ff56d7f2d657946e5c82e1c59fc#diff-ee126436faa9db1c57ae86d278387a97R120